src/style.css

```
@@ -1 +1,24 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  height: 16px;
+  width: 16px;
+  background-image: url('images/star.svg');
```

```suggestion
  height: 16px;
  width: 16px;
  margin-right: 4px;
  background-image: url('images/star.svg');
  background-position: center;
  background-repeat: no-repeat;
```

The order of your styles is really important for better readability